resources/js/Components/DashboardMetrics.vue

```diff
@@ -0,0 +1,43 @@
+<!-- component -->
+<!-- 
+    =======================================================================
+    Name    :   Dashboard Metrics - Default
+    Author  :   Surjith S M
+    Twitter :   @surjithctly
+
+    Other Variations Available: Stacked, Minimal, with icons
+
+    Get more components here 👉 https://web3templates.com/components
+
+    Copyright © 2021
+    =======================================================================
+ -->
+<template>
+	<div class="container max-w-6xl px-5 mx-auto">
+		<div class="grid gap-7 sm:grid-cols-2 lg:grid-cols-4">
+			<div class="p-5 bg-white rounded shadow-sm" v-for="(value, name) in metrics" :key="name">
+				<div class="text-base text-gray-400 ">{{name}}</div>
```

Comment:
could be a good idea to add the tailwind capitalize class here. basically a text decoration to force the first letter of each word to be capitalized. https://tailwindcss.com/docs/text-transform#capitalize

Reply:
Updated to have capitalization